### ISSUE TITLE
Make separate directory for Gridgen files

### ISSUE THREAD

**Issue by dbrakenhoff** [ArtesiaWater/nlmod#158]
I'd like Gridgen to dump its files in a separate directory, something like `model_ws/gridgen_ws`. This keeps the model_ws nice a clean with only mf6 files.

**Comment by rubencalje** [comment]
I agree, I would just put a folder named 'gridgen' in model_ws (so without the ws).

**Comment by rubencalje** [comment]
This is implemented in PR #174
